### PR TITLE
Add Skylark-related plans for Q3

### PR DESCRIPTION
**Pull request by laurentlb**
Next steps:
 - Link each line with a GitHub issue
 - Ask other subteams to create a similar page

### CODE REVIEW

**Comment by laurentlb** [comment]
I'd suggest to remove the menu on the left. See https://github.com/bazelbuild/bazel-website/pull/117#pullrequestreview-138286635

(I'll update this PR once the PR above has been submitted)

File: roadmaps/skylark.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: contribute
+title: Skylark roadmap
+---
+
+# Skylark Q3 2018 Roadmap
+
+*Last verified: July 18th, 2018*
```
Comment (laszlocsomor) [inline roadmaps/skylark.md]: The European in me (which is the only "-an" in me) really wants this to be [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (YYYY-MM-DD) date format, please.

**Comment by laszlocsomor** [comment]
Nice. I like the goals section!

File: roadmaps/skylark.md (hunk 1 above)
Comment (laurentlb) [reply roadmaps/skylark.md]: Me too :)
I copied the format from another doc, but I like YYYY-MM-DD much more.

Done

**Comment by laszlocsomor** [comment]
(I'll leave the merging to you.)

**Comment by laurentlb** [comment]
@laszlocsomor ptal

I'm adding it to the menu (since the question was not resolved yet and I'd like to submit now).

**Comment by laszlocsomor** [comment]
LGTM

**Pull request merged**